src/sentry/api/serializers/models/event.py

```diff
@@ -99,6 +99,11 @@ def serialize(self, obj, attrs, user):
             except TypeError:
                 received = None
 
+        event_type = obj.data.get('type', 'default')
+        metadata = obj.data.get('metadata') or {
+            'title': obj.message_short,
+        }
```

Comment:
I don't understand this comment, since the implementation here contradicts that view. If we're concerned with about upgrading the `metadata` field based on the inferred event type for events that weren't already upgraded on the way in, this isn't doing that.